incubator/sentry/templates/cron-deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-cron
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: {{ .Values.crenReplicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        role: cron
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}-cron
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args: ["run", "cron"]
+        ports:
+        - containerPort: {{ .Values.service.internalPort }}
+        env:
+        - name: SENTRY_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "fullname" . }}
+              key: sentry-secret
+        - name: SENTRY_DB_USER
+          value: {{ default "postgres" .Values.postgresUser | quote }}
```

Comment:
This isn't getting set in the pod? I manually set it in values but it's not being pulled in. Any ideas?